util.go

```diff
@@ -22,13 +22,25 @@ func fileExists(fname string) bool {
 	if err != nil {
 		return false
 	}
+	// file exists but might not be readable
+	f, err := os.Open(fname)
+	if err != nil {
+		log.Error("Cannot open file : [%v]", err)
```

Comment:
You should probably use `log.Errorf` here (similar to `fmt.Printf` vs. `fmt.Print`).